src/Microsoft.Azure.ServiceBus/Amqp/AmqpSubscriptionClient.cs

```diff
@@ -62,11 +64,14 @@ public MessageReceiver InnerReceiver
         /// <value>The number of messages that the subscription client can simultaneously request.</value>
         public int PrefetchCount
         {
-            get => this.ServiceBusConnection.PrefetchCount;
-
+            get => this.prefetchCount;
```

Comment:
>get => this.prefetchCount; [](start = 12, length = 26)

this doesnt need to maintain its own prefetch count. It could just pass the value to innerReceiver and calls innerReceiver.Prefetch count for get.